apps/crn-frontend/src/network/users/__tests__/UserProfile.test.tsx

```diff
@@ -451,6 +451,9 @@ it('navigates to the upcoming events tab', async () => {
     filters: new Set(),
     pageSize: 10,
     searchQuery: '',
-    userId: userResponse.id,
+    constraint: {
+      id: userResponse.id,
```

Comment:
This type seems a little limited. Shouldn't we support both type of constraints at the same time?

Reply:
No, there's no requirement to do that. 

If we start supporting both, we'll need to extend time building out the functionality to support both. i.e. what if both a teamId and a userId are passed through.

Decided as this wasn't a requirement, there's no need to do it.

Reply:
throw is both userId and teamId are passed

Reply:
done